### DISCUSSION TITLE
Torba içindeki toplar sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#179] [Soru Cevap]
Bir torbanın içinde tek bir top var. Bu topun rengini bilmiyoruz; bize yalnız onun siyah veya beyaz olabileceği söyleniyor.
Torbaya beyaz bir top atıyor ve karıştırıyoruz.
Sonra torbadan gelişigüzel bir top çekiyoruz, bakıyoruz ki top beyaz. Torbada kalan diğer topun da beyaz olma olasılığı nedir ?

**Comment by alicinar11** [comment]
En sonda torbada 2 toptan birinin beyaz olduğunu öğrendik torbada kalan diğer top zaten beyaz yada siyah olacaktır, yani torbada kalan diğer topunda beyaz olma olasılığı %50'dir.
